src/silx/gui/plot3d/items/mixins.py

```diff
@@ -61,11 +61,9 @@ class InterpolationMixIn(ItemMixInBase):
 
     def __init__(self, mode=NEAREST_INTERPOLATION, primitive=None):
         self.__primitive = primitive
+        self.__interpolationMode = mode
         self._syncPrimitiveInterpolation()
 
-        self.__interpolationMode = None
-        self.setInterpolation(mode)
-
```

Comment:
I am fine with all but this one seems strange to me. There is no need for calling `_syncPrimitiveInterpolation` or `_updated` in this case ?

Reply:
This is to fix an inheritance issue.
`_syncPrimitiveInterpolation` is already called and is needed to sync the `__primitive`.
And since it's in the `__init__`, there is no need to call `_updated` (= trigger a signal) because it's initializing it, and if I recall well, that was the issue: emitting a signal from a not fully initialized object.